package.json

```diff
@@ -26,7 +26,7 @@
     "lodash": "^4.17.4",
     "morgan": "^1.8.1",
     "pelias-logger": "^0.2.0",
-    "pelias-wof-admin-lookup": "^3.10.0",
+    "pelias-wof-admin-lookup": "pelias/wof-admin-lookup#refactor-for-layer-fallbacks",
```

Comment:
Github tells me a rebase is needed here. It will end up being `4.0.0` here, right?